### PR TITLE
config_gen-vrt-018.sh: Add gen-h-vrt-018 bash config file

### PR DESCRIPTION
**Pull request by ZiyadAtiyyeh**
This is due to adding new MARS setup for RH7.5

Signed-off-by: Ziyad Atiyyeh <ziyadat@mellanox.com>

### CODE REVIEW

File: config_gen-vrt-018.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+NIC=p1p1
+NIC2=p1p2
+VF=p1p1_0
+VF1=$VF
+VF2=p1p1_1
+REP=p1p1_0
```
Comment (roidayan) [inline config_gen-vrt-018.sh]: You set here VF and REP with the same name while you actually need them to be different interfaces.

Comment (roidayan) [reply config_gen-vrt-018.sh]: this is not a legit config file if VF and REP are the same name

Comment (ZiyadAtiyyeh) [reply config_gen-vrt-018.sh]: I updated the file.

**Comment by ZiyadAtiyyeh** [comment]
I updated the file.

**Pull request merged**